wpa_supplicant/ctrl_iface_zephyr.h

```diff
@@ -16,15 +16,36 @@
 #include "ctrl_iface.h"
 #include "common/wpa_ctrl.h"
 
+/**
+ * struct wpa_ctrl_mon - Data structure of control interface monitors
+ *
+ * This structure is used to store information about registered control
+ * interface monitors into struct wpa_supplicant.
+ */
+struct wpa_ctrl_mon {
+	struct wpa_ctrl_mon *next;
+	int sock;
+	int debug_level;
+	int errors;
+};
+
 struct ctrl_iface_priv {
 	struct wpa_supplicant *wpa_s;
 	/* 0 - wpa_cli, 1 - ctrl_iface */
 	int sock_pair[2];
+	int mon_sock_pair[2];
+	struct wpa_ctrl_mon *ctrl_dst;
 };
 
 struct ctrl_iface_global_priv {
 	struct wpa_global *global;
 	/* 0 - wpa_cli, 1 - ctrl_iface */
 	int sock_pair[2];
+	int mon_sock_pair[2];
+	struct wpa_ctrl_mon *ctrl_dst;
 };
 
+struct conn_msg {
+	char msg[256];
```

Comment:
better to have a macro for this and also any reason to choose only 256?

Reply:
Length was set to 256 based on a couple of logs whose length was close to 150 bytes.

Reply:
Done